### PR TITLE
Footer translatable

### PR DESCRIPTION
**Pull request by dcapillae**
Made footer's HTML translatable.

Closes #

### CODE REVIEW

File: openlibrary/templates/lib/nav_foot.html [hunk 1]
```diff
@@ -51,12 +51,12 @@ <h2>Help</h2>
...
+        <span>$:_('Open Library is an initiative of the <a href="//archive.org/">Internet Archive</a>, a 501(c)(3) non-profit, building a digital library of Internet sites and other cultural artifacts in  digital form. Other <a href="//archive.org/projects/">projects</a> include the <a href="//archive.org/web/">Wayback Machine</a>, <a href="//archive.org/">archive.org</a> and <a href="//archive-it.org">archive-it.org</a>')</span>
       </div>
       $ git_rev_hash = get_git_revision_short_hash()
       $if git_rev_hash:
         <div id="version-details">
-          <span class="version">version <a href="https://github.com/internetarchive/openlibrary/commit/$(git_rev_hash)">$git_rev_hash</a></span>
+          <span class="version">$:_('version <a href="https://github.com/internetarchive/openlibrary/commit/$(git_rev_hash)">$git_rev_hash</a>')</span>
```
Comment (mekarpeles) [inline openlibrary/templates/lib/nav_foot.html]: I think this embedding of the `$` could be problematic?

Comment (tfmorris) [reply openlibrary/templates/lib/nav_foot.html]: Yes, those references won't get evaluated correctly. Something like this is needed:

```
          <span class="version">$:_('version <a href="https://github.com/internetarchive/openlibrary/\
commit/%s">%s</a>', git_rev_hash, git_rev_hash)</span>

```

File: openlibrary/i18n/es/messages.po [hunk 2]
```diff
@@ -1664,7 +1664,7 @@ msgstr "Libro aleatorio"
 
 #: nav_head.html:15 nav_head.html:93
 msgid "Recent Community Edits"
-msgstr "Recientes ediciones de la comunidad"
+msgstr "Ediciones recientes de la comunidad"
```
Comment (tfmorris) [inline openlibrary/i18n/es/messages.po]: The changes in this file appear unrelated to the pull request. Please put them in a separate  PR.

File: openlibrary/templates/lib/nav_foot.html [hunk 3]
```diff
@@ -6,40 +6,40 @@
...
-        <h2>Develop</h2>
+        <h2>$:_('Develop')</h2>
         <ul>
-          <li><a href="/developers" title="$_('Explore Open Library Development Center')">Development Center</a></li>
-          <li><a href="/developers/api" title="$_('Explore Open Library APIs')">API Documentation</a></li>
-          <li><a href="/developers/dumps" title="$_('Bulk Open Library data')">Bulk Data Dumps</a></li>
+          <li><a href="/developers" title="$_('Explore Open Library Development Center')">$_('Development Center')</a></li>
+          <li><a href="/developers/api" title="$_('Explore Open Library APIs')">$_('API Documentation')</a></li>
+          <li><a href="/developers/dumps" title="$_('Bulk Open Library data')">$_('Bulk Data Dumps')</a></li>
           <li><a href="https://github.com/internetarchive/openlibrary/wiki/Writing-Bots" title="$_('Write a bot')">Writing Bots</a></li>
```
Comment (tfmorris) [inline openlibrary/templates/lib/nav_foot.html]: Line below needs "Writing Bots" translated.

Comment (dcapillae) [reply openlibrary/templates/lib/nav_foot.html]: Added the missing line. Thank you.

File: openlibrary/i18n/es/messages.po (hunk 2 above)
Comment (dcapillae) [reply openlibrary/i18n/es/messages.po]: Reverted changes not related to the PR. Thank you.

Comment (dcapillae) [reply openlibrary/i18n/es/messages.po]: @tfmorris, I'm not sure how this is done:
- Squash the typo cleanup commit into the main commit
- Rebase onto the current master to remove the merge commit

Comment (tfmorris) [reply openlibrary/i18n/es/messages.po]: Thanks for updating the PR. I'll take care of the rebasing it, but for the future here's how it's done. First we figure out how many commits we need to modify (in this case 7) and do an interactive rebase of them all.

`git rebase -i HEAD~7`

which gives you an editor session that looks like this:

```
pick eb8ab51 Made footer's HTML translatable
pick a18d382 Fixed some typos
pick 76b9883 Made footer's HTML translatable: added the missing translatable strings
pick 73c78b7 Fix git hash version i18n
pick 377e121 Added a missing translatable string
pick caf9507 Reverted unrelated changes to the PR
pick 23961be Clean up trailing whitespace

# Rebase ef7d551..23961be onto ef7d551 (7 commands)
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
# d, drop = remove commit
#
# These lines can be re-ordered; they are executed from top to bottom.
```
we can edit the buffer so that it looks like this:

```
pick eb8ab51 Made footer's HTML translatable
f 23961be Clean up trailing whitespace
f 76b9883 Made footer's HTML translatable: added the missing translatable strings
f 377e121 Added a missing translatable string
d a18d382 Fixed some typos
d caf9507 Reverted unrelated changes to the PR
pick 73c78b7 Fix git hash version i18n
```

which pulls all your commits to the template (plus one that I made to remove some trailing whitespace) together and merges them into a single commit. The two commits to the messages.po file we just drop since they counter each other.

Then to update the branch to the current master (and get rid of the merge commit) we do:

`git rebase master`

That leaves us with a branch containing just two commits which can be cleanly merged into the current master with no merge conflicts.
